DESCRIPTION

```diff
@@ -56,6 +56,7 @@ Suggests:
     ranger,
     rmarkdown,
     testthat,
+    tibble,
```

Comment:
We should not pull in tibble as a soft dep.
The following overloads the printer function for data.frame (can be extended for data.tables) and can live in everyone's `.Rprofile`:

```r
# tibble > data.frame
if (interactive() && "tibble" %in% rownames(utils::installed.packages())) {
  print.data.frame = function(x, ...) {
    tibble:::print.tbl(tibble::as_tibble(x), ...)
  }
}
```

Reply:
Doesn't CRAN give a warning/note when using triple colon in packages?

Reply:
This is not for inclusion in a package, this is for a local `.Rprofile`.